server/tstCompletion.ts

```diff
@@ -17,18 +17,84 @@ import {
   emptyChoiceData,
   getChoiceData,
 } from "./pythonUtilities";
+import { promisify } from "util";
+import { exec } from "child_process";
 
 function filterArray(currentArray: string[], whatToRemove: string) {
   return currentArray.filter((element) => element !== whatToRemove);
 }
 
+/**
+ * Checks if a specific VectorCAST option is set to a given value in the specified environment.
+ *
+ * @param {string} enviroPath - The file path to the environment where the command should be executed.
+ * @param {string} option - The VectorCAST option to check (e.g., 'VCAST_CODED_TESTS_SUPPORT').
+ * @param {string} optionValue - The expected value of the option (e.g., 'True', ...).
+ * @returns {Promise<boolean>} - Resolves to `true` if the option value matches, otherwise `false`.
+ */
+export async function checkClicastOption(
```

Comment:
A couple of things to consider here:
- https://github.com/vectorgrp/vector-vscode-vcast/blob/2cb2dc5ccedaa9e507174b2d2e47befce2e7788a/src/vcastInstallation.ts#L304 is definining the `clicastCommandToUse`
- https://github.com/vectorgrp/vector-vscode-vcast/blob/2cb2dc5ccedaa9e507174b2d2e47befce2e7788a/src/vcastAdapter.ts#L244 is an example 
 - you should try to re-use existing functions/keep the consistency in the repo
 - In the code above you can see ` executeCommandSync` being used for a very similar command (server variant is https://github.com/vectorgrp/vector-vscode-vcast/blob/2cb2dc5ccedaa9e507174b2d2e47befce2e7788a/src/vcastAdapter.ts#L219C27-L219C33)
 - If we are calling this for every completion, some caching might make sense
   - Something like https://www.npmjs.com/package/@type-cacheable/core with lru cache